bitrise.yml

```diff
@@ -12,50 +12,144 @@ workflows:
     - errcheck:
     - go-test:
     after_run:
-    - apk_deploy_test
+    - apk_deploy_test_1
```

Comment:
Can you please add a comment that apk_deploy_test_2 is supposed to be run after apk_deploy_test_1, what it validates, and how is triggered or any change is needed to be wired in?

Reply:
Added comment.